lib/editor/actions/editor.js

```diff
@@ -183,15 +188,16 @@ export function stopLockTimer () {
 function maintainEditorLock (
   sessionId: string,
   feedId: string,
+  itemToLock: string,
   timer?: IntervalID
 ) {
   return function (dispatch: dispatchFn, getState: getStateFn) {
-    const url = `/api/editor/secure/lock/${sessionId}?feedId=${feedId}`
+    const url = `${LOCK_API_PATH}/${sessionId}?feedId=${feedId}&item=${itemToLock}`
     // Note: this action is called only when the browser tab/window is visible.
```

Comment:
I'm having trouble seeing how this ties in to the new component which seems to manage visibility. Is this definitely still true?

Reply:
OK, I'm seeing that the new component disables the timer when the page isn't visible, so I think I get it now!